examples/server.rs

```diff
@@ -34,8 +34,8 @@ fn main() {
 
     let neighbor_cache = NeighborCache::new(BTreeMap::new());
 
-    let udp_rx_buffer = UdpSocketBuffer::new(vec![UdpPacketBuffer::new(vec![0; 64])]);
-    let udp_tx_buffer = UdpSocketBuffer::new(vec![UdpPacketBuffer::new(vec![0; 128])]);
+    let udp_rx_buffer = UdpSocketBuffer::new(vec![Default::default()], vec![0; 64]);
```

Comment:
I would prefer explicit initialization in *examples*, to make it clear what is being constructed.